README.md

```diff
@@ -71,8 +71,8 @@ methods.
 
 You can interact with QuestDB using the following interfaces:
 
-- [Web Console](https://questdb.io/docs/develop/web-console/) listening on
-  port `9000`
+- [Web Console](https://questdb.io/docs/develop/web-console/) listening on port
```

Comment:
hm, this is strange. I don't remember doing this change. is there a chance this was done automatically by a commit hook?

Reply:
I mean - how come my own commit - the one you are just reviewing - has this change? 
it's not important, it's just a white space. chances are I pressed a key accidentally.